doc/en/user/source/community/geofence/cache.rst

```diff
@@ -0,0 +1,48 @@
+.. _geofence_cache:
+
+GeoFence Cache REST
+===================
+
+The Geofence client cache can be queried and invalidated through a REST service.
+
+Requests
+--------
+
+``/geofence/ruleCache/info``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Retrieve information about the geofence caches.
```

Comment:
cache singular?

Reply:
there are three caches: rule cache, admin cache and user cache